python/ray/data/tests/test_binary.py

```diff
@@ -195,6 +198,64 @@ def skip_unpartitioned(kv_dict):
     )
 
 
+def _gen_chunked_binary(
+    dir_path: str, total_size: int, max_file_size: Optional[int] = None
+):
+    chunk_size = max_file_size or 256 * MiB
+    num_chunks = total_size // chunk_size
+    remainder = total_size % chunk_size
+
+    if max_file_size is not None and max_file_size < total_size:
+        for i in range(num_chunks):
+            filename = f"part_{i}.bin"
+            with open(f"{dir_path}/{filename}", "wb") as f:
+                f.write(b"a" * chunk_size)
+
+                print(f">>> Written file: {filename}")
+
+    else:
+        with open(f"{dir_path}/chunk.bin", "wb") as f:
+            for i in range(num_chunks):
+                f.write(b"a" * chunk_size)
+
+                print(f">>> Written chunk #{i}")
```

Comment:
How does writing the data in chunks affect the behavior of Ray Data?

Reply:
It just makes test faster alas does not require 2Gb of heap/object store memory (for a single chunk)

Reply:
Got it. Could we add a note about that? It wasn't obvious that this isn't directly related to the test